Docs audit and updates. (#202)
patternfly/virtual-assistant: packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/demos/Chatbot.md

> 2. A [`<ChatbotHeader>`](/patternfly-ai/chatbot/chatbot-header) with all built sub-components laid out, including a `<ChatbotHeaderTitle>` that changes its presentation depending on the display mode.

Reviewer: We want to adjust the link in line 59 to be  /patternfly-ai/chatbot/chatbot-container if we're renaming it.

3. The ability to swap display modes via `<ChatbotHeaderOptionsDropdown>`

> 4. [`<ChatbotContent>` and `<MessageBox>`](/patternfly-ai/chatbot/chatbot#chatbot-content-and-message-box) with:

Reviewer: ```suggestion
4. [`<ChatbotContent>` and `<MessageBox>`](/patternfly-ai/chatbot/chatbot-container#content-and-message-box) with:
```

  - A `<ChatbotWelcomePrompt>`
  - An initial [user `<Message>`](/patternfly-ai/chatbot/chatbot-messages) and an initial bot message with [response actions.](/patternfly-ai/chatbot/chatbot-messages/#messages-actions)
  - Logic for enabling auto-scrolling to the most recent message whenever a new message is sent or received using a `scrollToBottomRef`
5. A [`<ChatbotFooter>`](/patternfly-ai/chatbot/chatbot-footer) with a [`<ChatbotFootNote>`](/patternfly-ai/chatbot/chatbot-footer#footnote-with-popover) and a `<MessageBar>` that contains the abilities of:

>   - [Speech to text.](/patternfly-ai/chatbot/chatbot-footer#message-bar-with-speech-to-text)

Reviewer: ```suggestion
  - [Speech to text.](/patternfly-ai/chatbot/chatbot-footerr#message-bar-with-speech-recognition-and-file-attachment)
```

  - Sending a message to the chatbot.
  - Receiving a response from a backend AI tool with a loading message state.

> 6. A [`<ChatbotConversationHistoryNav>`](/patternfly-ai/chatbot/chatbot-conversation-history-nav) toggled open and closed by the `<ChatbotHeaderMenu`> in the `<ChatbotHeader>`.

Reviewer: ```suggestion
6. A [`<ChatbotConversationHistoryNav>`](/patternfly-ai/chatbot/chatbot-conversation-history) toggled open and closed by the `<ChatbotHeaderMenu`> in the `<ChatbotHeader>`.
```
